data/raw/cpm2014_partners.json

```diff
@@ -0,0 +1,128 @@
+[
+  {
+    "name_be": "Where Minsk",
+    "name_ru": "Where Minsk",
+    "name_en": "Where Minsk",
+    "link": "http://www.spn.ru/publishing/whereminsk/",
+    "image": "images/where.png"
+  },
+  {
+    "name_be": "relax.by - развлечения в Минске, развлекательные центры столицы",
+    "name_ru": "relax.by - развлечения в Минске, развлекательные центры столицы",
+    "name_en": "relax.by - развлечения в Минске, развлекательные центры столицы",
+    "link": "http://relax.by/",
+    "image": "images/relax.png"
+  },
+  {
+    "name_be": "Минск 24 ДОК",
+    "name_ru": "Минск 24 ДОК",
+    "name_en": "Минск 24 ДОК",
+    "link": "http://mtis.tv/",
+    "image": "images/minsk24dok.png"
+  },
+  {
+    "name_be": "Nomadic Film Club",
+    "name_ru": "Nomadic Film Club",
+    "name_en": "Nomadic Film Club",
+    "link": "http://vagrant.kinaklub.org/",
+    "image": "images/nomadic.png"
+  },
+  {
+    "name_be": "Минская Киношкола-студия",
+    "name_ru": "Минская Киношкола-студия",
+    "name_en": "Минская Киношкола-студия",
```

Comment:
That's not translated. Is there a strategy to get this done? Should these be left blank then?